File: webscraping/18-1-20/codigo_uni.js
function obtenerLetraCodigoUNI(codigoUNI /*solo números*/) {
    let cod = String(codigoUNI);
    let sum = 0;
    const arr = [2, 1, 2, 3, 4, 5, 6, 7];
    for (let i = 0; i < cod.length; i++) {
        sum += cod.charAt(i) * arr[i];
    }
    return String.fromCharCode(65 + sum % 11);
}

console.log(obtenerLetraCodigoUNI(20151216));

File: webscraping/18-1-20 clase/codigo_uni.py
def obtenerLetraCodigoUNI(codigoUNI):
    cod = str(codigoUNI)
    arr = [2,1,2,3,4,5,6,7]
    suma = 0
    for i in range(len(cod)):
        suma += int(cod[i]) * arr[i]
    return chr(65 + suma % 11)

print(obtenerLetraCodigoUNI(20151216))
